plugins/event_notification/providers/email/lib/batch/recipients/KEMailNotificationRecipientEngine.php

```diff
@@ -36,4 +36,64 @@ public static function getEmailNotificationRecipientEngine(KalturaEmailNotificat
 	 * @param array $contentParameters
 	 */
 	abstract function getRecipients (array $contentParameters);
+
+	protected function getUsersByUserIds($userIds)
+	{
+		$userFilter = new KalturaUserFilter();
+		$pager = new KalturaFilterPager();
+		$maxIdIn = 150;
+		$pager->pageSize = $maxIdIn;
+		$idsArray = explode(',', $userIds);
+		$offset = 0;
+		$usersListResponse = array();
+		$allUsers = array();
+
+		while($offset * $maxIdIn < count($idsArray))
```

Comment:
the condition is a bit complicated, why not having a counter that will count the handled ids?

Reply:
Changed the condition to a counter.